internal/handlers/handler.go

```
@@ -133,43 +136,70 @@ func (h *Handler) Handle(ctx context.Context, reqHeader *wire.MsgHeader, reqBody
 		fallthrough
 	default:
 		requests.WithLabelValues(command).Inc()
-		panic(fmt.Sprintf("unexpected OpCode %s", reqHeader.OpCode))
+
+		err = lazyerrors.Errorf("unexpected OpCode %s", reqHeader.OpCode)
```

is the OpCode is unexpected, return here?

---

no I see now - it's to be processed.
But the error is not handled somehow, it's overwritten by `	b, err := resBody.MarshalBinary()` later on

---

It is handled a few lines below

---

ok